recipes/aaf/all/conanfile.py

```diff
@@ -0,0 +1,91 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AafConan(ConanFile):
+    name = "aaf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/aaf/"
+    description = "A  cross-platform SDK for AAF. AAF is a metadata management system and file format for use in professional multimedia creation and authoring."
+    topics = ("aaf", "multimedia", "crossplatform")
+    license = "AAFSDKPSL-2.0"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "structuredstorage": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "structuredstorage": False,
+    }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("expat/2.4.1")
+        self.requires("libjpeg/9d")
+        if self.settings.os in ("FreeBSD", "Linux"):
+            self.requires("libuuid/1.0.3")
+
+    def validate(self):
+        if self.settings.os == "Macos" and self.settings.arch == "armv8":
+            raise ConanInvalidConfiguration("ARM v8 not supported")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def build(self):
+        for patch in self.conan_data.get("patches", {}).get(self.version, []):
+            tools.patch(**patch)
+
+        cmake = CMake(self)
+
+        if tools.is_apple_os(self.settings.os):
+            cmake.definitions["PLATFORM"] = "apple-clang"
+        elif self.settings.compiler == "Visual Studio":
+            cmake.definitions["PLATFORM"] = "vc"
+        else:
+            cmake.definitions["PLATFORM"] = self.settings.os
+
+        cmake.definitions["ARCH"] = "x86_64"  # ARCH is used only for setting the output directory. So itsvalue does not matter here.
+        cmake.definitions["AAF_NO_STRUCTURED_STORAGE"] = not self.options.structuredstorage
```

Comment:
This is just my OCD having done a few too many code reviews today
↪️ So this is completely optional

But it would be nice to throw in an `_` in the option

```suggestion
        cmake.definitions["AAF_NO_STRUCTURED_STORAGE"] = not self.options.structured_storage
```